modules/core/src/bitgo.ts

```diff
@@ -607,10 +607,11 @@ export class BitGo {
 
   /**
    * Create a basecoin object
-   * @param coinName
+   * @param coinIdentifier
+   * @param networkName
    */
-  coin(coinName: string): BaseCoin {
-    return GlobalCoinFactory.getInstance(this, coinName);
+  coin(coinIdentifier: string, networkName: string | null | undefined = undefined): BaseCoin {
```

Comment:
why are we renaming to identifier? isnt it still a name (i.e usdt)

Reply:
also like name bc it lets us distinguish whether the "identifier" is a name or an address